core/commands/partials/vagrant_configurator.rb

```diff
@@ -57,13 +58,24 @@ def node_provisioned?(node, logger)
     end
   end
 
-  # Split list of nodes between running and halt ones
+  # Check that specified node is brought up.
   #
-  # @param nodes [Array<String] name of nodes to check
+  # @param node [String] name of node that should be checked.
   # @param logger [Out] logger to log information to.
-  # @return [Array<String>, Array<String>] nodes that are running and those that are not
-  def running_and_halt_nodes(nodes, logger)
-    nodes.partition { |node| VagrantCommands.node_running?(node, logger) }
+  # @return [Bool] true if node needs to be re-created.
+  def broken_node?(node, logger)
```

Comment:
Methods `broken_node?` and  `unconfigured_node?` are no longer used. You have messed-up the merge.